internal/connect/telemetry.go

```diff
@@ -0,0 +1,43 @@
+package connect
+
+import (
+	"log/slog"
+	"path/filepath"
+
+	"github.com/SUSE/connect-ng/internal/util"
+	"github.com/SUSE/telemetry"
+	"github.com/SUSE/telemetry/pkg/client"
+)
+
+func pushTelemetry(content []byte) {
+	// This configuration file is to be provided by an external package.
```

Comment:
This can be better handled using the [Status()](https://github.com/SUSE/telemetry/blob/e727356476d90c2976cbd9ed41d272175a19e371/telemetry.go#L169) interface call, with a check like:

```
        if telemetry.Status() != telemetry.CLIENT_REGISTERED {
                return
        }
```

This will skip submitting telemetry until the telemetry client is able to register with the upstream telemetry server.

Or if you want the equivalent of this check it would be:

```
        if telemetry.Status() != telemetry.CLIENT_CONFIG_ACCESSIBLE {
                return
        }
```